test/system/errorService/service.js

```diff
@@ -14,12 +14,11 @@
 
 'use strict'
 
-var mu = require('../../../core/core')()
-var mue = require('mu-error')()
+var mu = require('../../../packages/mu')()
```

Comment:
This just looks ugly, how about

```js
var mu = require('../../mu')()
```

given that this is already inside packages?